test/unit/apps/rotm/controllers/report.js

```diff
@@ -20,6 +23,10 @@ describe('Report Controller', () => {
     controller.should.have.property('locals').that.is.a('function');
   });
 
+  it('has a validateField method', () => {
+    controller.should.have.property('validateField').that.is.a('function');
```

Comment:
@JoeChapman there would be no difference. What Lenny was saying is if you end a chain with a property, not a method, and the property is undefined, then nothing is thrown. However if you end with a function (like here) if the function is undefined, it will throw an error as you have tried to call an undefined function.